.npmignore

```diff
@@ -0,0 +1,14 @@
+# Ignore the directory
```

Comment:
Look at this one:
https://github.com/pact-foundation/pact-js/blob/master/.npmignore

I think that for our template, as we need a lot of dev files due to its a template to help development, we could exclude these files:

.github
https://github.com/rubemfsv/clean-react-app/blob/main/CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE.md
CHANGELOG.md
node_modules
.vscode
coverage
dist
package-lock.json
public/js
*.DS_Store
yarn.lock

Reply:
Hi @rubemfsv, I fixed all the issues in the new PR, I'm closing this one. https://github.com/rubemfsv/clean-react-app/pull/42